src/common/validation/validate-url.ts

```diff
@@ -0,0 +1,5 @@
+import { isWebUri } from 'valid-url';
+
+export function isValidUrl(str: string) {
```

Comment:
thoughts on using a simple validation function instead of introducing a new library, something like:
```
const isValidUrl = (url: string) => {
  try {
    new URL(url);
  } catch (e) {
    return false;
  }
  return true;
};
```

Reply:
`new URL("javascript:alert('lskdjfs')")` doesn't throw ⚠️

Reply:
Indeed, we should do more than just using new URL() as @kyranjamie pointed out.
I introduce third party libraries with care and I think it's justified here as it has no dependencies, is small and easily [verifiable](https://github.com/ogt/valid-url/blob/master/index.js), has a single purpose and the url format in general does not change.